pyvista/core/dataset.py

```diff
@@ -2568,3 +2575,61 @@ def cell_point_ids(self, ind: int) -> List[int]:
         cell = self.GetCell(ind)
         point_ids = cell.GetPointIds()
         return [point_ids.GetId(i) for i in range(point_ids.GetNumberOfIds())]
+
+    def point_is_inside_cell(
+        self, ind: int, point: Union[VectorArray, NumericArray]
```

Comment:
How strong feelings do you have about having the cell index first and the point(s) second? I'm wondering if we should flip the order. Or are there other similar filters that this might match?

Reply:
I chose this since this is a cell wise operation, which feels like it should be the first parameter to me, but I'm open to other opinions.  I like the idea of an eventual API like `dataset.cells[10].is_point_inside(points)`, which would suggest the current order for now.  But there was also the suggestion of being able to provide a Sequence or similar of cells to this type of function.

Reply:
OK, let's leave this as is.